include/VoidAccessor.h

```diff
@@ -0,0 +1,90 @@
+#pragma once
+
+#include <string>
+
+#include <ChimeraTK/VoidRegisterAccessor.h>
+
+#include "Application.h"
+#include "InversionOfControlAccessor.h"
+
+namespace ChimeraTK {
+
+  /********************************************************************************************************************/
+
+  /** Accessor for void variables (i.e. no data, just the "trigger" information). Note for users: Use the
+   * convenience classes VoidInput and VoidOutput instead of this class directly. */
+  class VoidAccessor : public ChimeraTK::VoidRegisterAccessor, public InversionOfControlAccessor<VoidAccessor> {
+   public:
+    using InversionOfControlAccessor<VoidAccessor>::operator VariableNetworkNode;
+    using InversionOfControlAccessor<VoidAccessor>::operator>>;
+    void replace(const ChimeraTK::NDRegisterAccessorAbstractor<ChimeraTK::Void>& newAccessor) = delete;
+    using InversionOfControlAccessor<VoidAccessor>::replace;
+    VoidAccessor& operator=(VoidAccessor& other) = delete;
+    using ChimeraTK::VoidRegisterAccessor::operator=;
+
+    /** Move constructor */
+    VoidAccessor(VoidAccessor&& other) { InversionOfControlAccessor<VoidAccessor>::replace(std::move(other)); }
+
+    /** Move assignment. */
+    VoidAccessor& operator=(VoidAccessor&& other) {
+      // Having a move-assignment operator is required to use the move-assignment
+      // operator of a module containing an accessor.
+      InversionOfControlAccessor<VoidAccessor>::replace(std::move(other));
+      return *this;
+    }
+
```

Comment:
Similar for this part, identical code exists in Array and Scalar. I'd add a ticket for refactor here.

Reply:
same ticket as above